### PR TITLE
KOFAMSCAN

### PR DESCRIPTION
**Pull request by SilasK**
**Replace this sentence with a detailed description of your pull request**

----

> Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [x] This PR adds a new recipe.
* [ ] AFAIK, this recipe **is directly relevant to the biological sciences**
      (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

> Everyone has access to the following BiocondaBot commands, which can be given in a comment:
>
>  * `@BiocondaBot please update` will cause the BiocondaBot to merge the master branch into a PR
>  * `@BiocondaBot please add label` will add the `please review & merge` label.
>  * `@BiocondaBot please fetch artifacts` will post links to packages and docker containers built by the CI system. You can use this to test packages locally before merging.
>
> For members of the Bioconda project, the following command is also available:
>
>  * `@BiocondaBot please merge` will cause packages/containers to be uploaded and a PR merged. Someone must approve a PR first! This has the benefit of not wasting CI build time required by manually merging PRs.
>
> If you have questions, please post them in gitter or ping `@bioconda/core` in a comment (if you are not able to directly ping `@bioconda/core` then the bot will repost your comment and enable pinging).

### CODE REVIEW

**Comment by SilasK** [comment]
@BiocondaBot please update

File: recipes/kofamscan/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+source:
+
+  url: ftp://ftp.genome.jp/pub/tools/kofamscan/archives/kofamscan-{{ version }}/kofamscan-{{ version }}.tar.gz
+  sha256: 8d039abc69cc5671c35da8d41c8b48f742190e0b4002b76989e52afe82e963e5
+  # - folder: db
+  #   url: ftp://ftp.genome.jp/pub/db/kofam/archives/{{ db_version }}/ko_list.gz
+  #   sha256: 5a642fdb442e0012b9de37cc513179161aa3548c5a47d3b0306fa5d07a4b3f85
+  # - folder: db
+  #   url: ftp://ftp.genome.jp/pub/db/kofam/archives/{{ db_version }}/profiles.tar.gz
+  #   sha256: 27e1d4486daaa7b20d21734cd7589a0e11104186cf5ae5b1062785c26423fe91
```
Comment (dpryan79) [inline recipes/kofamscan/meta.yaml]: Are these going to be needed?

**Comment by dpryan79** [comment]
@bioconda-bot please fetch artifacts

File: recipes/kofamscan/meta.yaml (hunk 1 above)
Comment (SilasK) [reply recipes/kofamscan/meta.yaml]: c009e1b

**Comment by SilasK** [comment]
@dpryan79 
@BiocondaBot please add label

**Comment by dpryan79** [comment]
@bioconda-bot please merge

**Pull request merged**